packages/node_modules/@webex/plugin-meetings/src/roap/state.js

```diff
@@ -25,6 +25,10 @@ const shouldStep = (roap, meeting) => {
 };
 
 const handleTransition = (value, signal, meeting) => {
+  LoggerProxy.logger.log('Roap:state#handleTransition --> current roap state', value);
+  LoggerProxy.logger.log('Roap:state#handleTransition --> next roap state', signal);
```

Comment:
log text is incorrect - this is not displaying the next state, but the received signal
also a minor thing, but wouldn't it be better to log both in a single log call?